tests/contrib/django/test_django_dbm.py

```diff
@@ -0,0 +1,95 @@
+from django.db import connections
+import mock
+
+from ddtrace import Pin
+from ddtrace.internal import core
+from ddtrace.propagation._database_monitoring import handle_dbm_injection
+from tests.contrib import shared_tests
+from tests.utils import DummyTracer
+from tests.utils import override_config
+from tests.utils import override_dbm_config
+from tests.utils import override_env
+from tests.utils import override_global_config
+
+from ...contrib.config import POSTGRES_CONFIG
+
+
+core.on("django-postgres.execute", handle_dbm_injection, "result")
```

Comment:
why do we do this here? don't we want to actually enable DBM to ensure the core events work as expected?

Reply:
Enabling DBM is kinda weird and needs to happen at tracer `init`. During tracer `init`, we add these core hooks. Since we don't want DBM enabled for every django test, we need to manually add the hook. I'll add a comment explaining why we are doing this above.

Reply:
removed